script/deploy/facets/DeployTokenWrapper.s.sol

```diff
@@ -29,6 +29,19 @@ contract DeployScript is DeployScriptBase {
             string.concat(".", network, ".wrappedNativeAddress")
         );
 
+        // try to get converter address, default to address(0) if not found
```

Comment:
this means if there is an error in the JSON we deploy with address(0) despite wanting to deploy something with a different address. I dont think this is in our interest. 
Since the config file has a `converterAddress` value for supported network, we dont really need a try/catch here imho.

Instead, we can just read the address from the config file and if it's `!= address(0)` then I believe we should make sure it's a contract address. Wdyt?

Reply:
Fixed dcb8a6d1, 8843234d: Added converter contract validation